README.md

```diff
@@ -304,6 +304,7 @@ Table of Contents
   * [Prismic](https://www.prismic.io/) — Headless CMS. Content management interface with fully hosted and scalable API. The Community Plan provides 1 user with unlimited API calls, documents, custom types, assets, and locales. Everything that you need for your next project. Bigger free plans available for Open Content/Open Source projects.
   * [sanity.io](https://www.sanity.io/) – Hosted backend for structured content with customizable MIT licensed editor built with React. Unlimited projects. 3 users, 2 datasets, 500k API CDN requests, 5GB assets for free per project
   * [sensenet](https://sensenet.com) - API-first headless CMS providing enterprise-grade solutions for businesses of all size. The Developer plan provides 3 users, 500 content items, 3 built-in roles, 25+5 content types, fully accessible REST API, document preview generation and Office Online editing.
+  * [Strapi](https://strapi.io/) - RESTful or GraphQL headless CMS fully customizable and developer-first.
```

Comment:
Isn't this self hosted only?

Reply:
yes it is but I thought it a very helpful headless cms tool

Reply:
Sorry, the list is what it is, there are an infinite of helpful things that isn't on the list, I can't add them all - so we have to focus on the purpose of the list.

thank you!